tests/unit/api/c/source/test_source.cpp

```diff
@@ -57,15 +72,35 @@
     EXPECT_NE(sacn_lock_fake.call_count, previous_lock_count);                \
     EXPECT_EQ(sacn_lock_fake.call_count, sacn_unlock_fake.call_count);        \
   } while (0)
+#define VERIFY_NO_LOCKING_AND_RETURN_VALUE(function_call, expected_return_value) \
+  do                                                                             \
+  {                                                                              \
+    unsigned int previous_lock_count = sacn_lock_fake.call_count;                \
+    EXPECT_EQ(function_call, expected_return_value);                             \
+    EXPECT_EQ(sacn_lock_fake.call_count, previous_lock_count);                   \
+    EXPECT_EQ(sacn_lock_fake.call_count, sacn_unlock_fake.call_count);           \
+  } while (0)
 
 static const etcpal::Uuid kTestLocalCid = etcpal::Uuid::FromString("5103d586-44bf-46df-8c5a-e690f3dd6e22");
 static const std::string kTestLocalName = std::string("Test Source");
```

Comment:
These don't need to be std::strings, since you use c_str() to convert them back to const char* anyway.
```cpp
constexpr char kTestLocalName[] = "Test Source";
```

However, if you ever do need to define std::strings at file scope like this, just assign the string literal to them for more concise syntax:
```cpp
static const std::string kTestLocalName = "Test Source";
```

Reply:
https://github.com/ETCLabs/sACN/pull/47/commits/578a65f739891244091905dd93959c063d51a994